app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/ConnectedServers.tsx

```diff
@@ -146,12 +181,32 @@ const ConnectedServersTable = ({ jenkinsServerList, refreshServers }: ConnectedS
 			refreshServers();
 		}
 
+		await analyticsClient.sendAnalytics(
+			AnalyticsEventTypes.UiEvent,
+			AnalyticsUiEventsEnum.DisconnectServerConfirmConfiguredStateName,
+			{
+				source: AnalyticsScreenEventsEnum.ConfigurationConfiguredStateScreenName,
+				action: 'clicked disconnect',
+				actionSubject: 'button'
+			}
+		);
+
 		closeConfirmServerDisconnect();
 	};
 
-	const closeConfirmServerDisconnect = () => {
+	const closeConfirmServerDisconnect = async () => {
 		setShowConfirmServerDisconnect(false);
 		setIsLoading(false);
+
+		await analyticsClient.sendAnalytics(
+			AnalyticsEventTypes.UiEvent,
+			AnalyticsUiEventsEnum.DisconnectServerModalClosedConfiguredStateName,
+			{
+				source: AnalyticsScreenEventsEnum.ConfigurationConfiguredStateScreenName,
+				action: 'clicked cancel or disconnected server',
```

Comment:
Having `or` could be confusing when analyzing data. This method is triggered on the dismissal of the modal, so should be just `clicked cancel`.

Reply:
Then I think it should only be `dismissed the modal`. 
We already have events sent when the user cancels or disconnects right, then if we want to get both of the above scenarios, we first check if the user has clicked cancel or disconnect followed by this dismissal action.